app/__init__.py

```diff
@@ -0,0 +1,18 @@
+import sentry_sdk
+from django.conf import settings
+from sentry_sdk.integrations.django import DjangoIntegration
+
+if settings.SENTRY_DSN:
+    sentry_sdk.init(
+        dsn=settings.SENTRY_DSN,
+        environment=settings.ENVIRONMENT_NAME,
+        release=(
+            f"ds-search@{settings.BUILD_VERSION}"
+            if settings.BUILD_VERSION
+            else ""
+        ),
```

Comment:
```suggestion
        release=settings.BUILD_VERSION,
```

The release should be a proper version number (semver/calver) to allow Sentry to properly track fixes.